docs/core/seeding.rst

```diff
@@ -324,6 +425,12 @@ location. The ordering by location is only done to make sure reimplementations
 (such as the GPU code) are comparable and return the bitwise exactly same
 result.
 
+When a seed is accepted and seed confirmation is enabled, the weight of that seed
+is assigned to each of its SPs. Each SP will hold the weight of the best seed that
+includes that SP. This information is used in the selection of the next seeds:
+The seed is kept only if its weight is greater than the weight of at least one of
```

Comment:
Maybe start by saying that it is some sort of voting scheme that is implemented here. BTW. Greater or equal, right? 
I.e. in the illustrations above there will be only one weight value.